gammapy/irf/core.py

```diff
@@ -958,9 +958,10 @@ def cutout(self, position, width, mode="trim"):
         cutout : `IRFMap`
             Cutout IRF map.
         """
-        irf_map = self._irf_map.cutout(position, width, mode)
+
+        irf_map = self._irf_map.cutout(position, width, mode, min_npix=3)
```

Comment:
I think the option should be explicitly available on the `cutout` arguments. A minima it should be explained that the minimum cutout is 3 pixels wide.

Reply:
done